samples/GUIDE.md

```diff
@@ -74,6 +74,40 @@ Notes:
 
 * *RETURN_VALUE* is either null, if a DSL does not have a return value, or the returnable object of the DSL, if it does
 
+### Optional parameters
+
+Any parameter where the name starts with the word `optional_`, can be omitted from requests. For example, `post-with-optional.yml` has one required parameter 
+and one 
+`optional_` type parameter.
```

Comment:
```suggestion
Any parameter where the name starts with the word `optional_`, can be omitted from requests. For example, `post-with-optional.yml` has one required parameter 
and one 
`optional_` type parameter.
```

Using this kind of prefix should be replaced in later stages but will pass it at the moment.